### ISSUE TITLE
Information about training option in torch.onnx.export()

### ISSUE THREAD

**Issue by alex96295** [onnx/onnx-tensorflow#667]
Hi,

I would like to ask a question: if I happen to have a pytorch .pth model, and want to convert it in onnx, then I would go for torch.onnx library. Yet, I want to convert the onnx model to tensorflow model. 

**First question**: should I put model.eval() before exporting? Because there is an option in torch.onnx.export() - called 'training' - that says: 

``` training (bool, default False): export the model in training mode.  At the moment, ONNX is oriented towards exporting models for inference only, so you will generally not need to set this to True. ```

Therefore, I was wondering how that option is related to me putting or no 'model.eval()' before.

**Second question**: If however, I want to convert the onnx model to Caffe, I noticed a repo able to do this. Provided that it works (could it not? I mean is it 'not-standard' to do?) since I guess the repo is not official like this repo, I don't know much about Caffe, and only that I would need a .prototxt file and a .caffemodel file. The question is:

- For converting onnx to caffe, should the model be in 'inference' mode or not? I mean, there exists in Caffe the same differences as for Tensorflow, Keras and Pytorch where you have to detach the training mode from the evaluation mode (right?)

Thank you!

**Comment by chinhuang007** [comment]
Looks like your questions should go https://github.com/pytorch/pytorch. This repo is to convert onnx models to tensorflow.

Just to answer second question in general, the current onnx models are for inference only. So the conversion to Caffe or Tensorflow doesn't have to deal with training.

**Comment by alex96295** [comment]
@chinhuang007 ok thank you for the answer :)
